packages/react-components/react-spinbutton/src/stories/SpinButton/SpinButtonAccessibility.md

```diff
@@ -0,0 +1,100 @@
+## Component features and behavior
+
+This section documents accessibility-related behavoirs of spin buttons.
+
+### Keyboarding
+
+#### Tab Order
+1. Value field
+
+#### States
+1. Rest (Focused)
+2. Editing (Focused and editing)
+
+#### Keyboard State Diagram
+
+| Starting state | Transition | Resulting state |
+| ---------- | ------------ | ------- |
+| Content before spin button | Tab | Rest |
+| Rest  | Tab | Content after spin button |
+| Editing  | Tab | Content after spin button (Value committed) |
+| Editing  | Enter | Rest (Value committed)|
+| Rest  | Any Edit Key (that results in a change) | Editing |
+| Editing  | Any Edit Key | Editing |
+| Content after spin button (Value committed) | Shift + Tab | Rest |
+
+##### Edit keys
+| Edit Key | Result |
+| ---------- | ------------ |
+| Home | First item in defined range |
+| End | Last item in defined range |
+| Up arrow | Increments value higher, based on the step prop (defaulting to 1) |
+| Down arrow | Increments value lower, based on the step prop (defaulting to 1) |
+| Page up | Increments value higher, based on the stepPage prop (defaulting to 1) |
+| Page down | Increments value lower, based on the stepPage prop (defaulting to 1) |
+| Typing valid value | Valid value |
+
+### High contrast mode
+
+Be careful about visibility of the spin button arrows in high contrast mode.
+TODO: Either add specifics about SVGs in high contrast and how to customize the icon color without messing up HCM or (ideally) link to the Icon section on high contrast, when it exists.
+
+### Motion and animation
+
+There is an animation when this element receives focus. This respects reduced-motion media queries.
+
+When a user holds down an edit key or mouse click, there is an animation of spinning numbers to provide a sense of how fast the value is changing. This animation is an interaction cue and therefore is acceptable even when a user has set a preference for reduced motion.
+
+### Semantics
+
+## Known issues
+
+Placeholder: If there are any current known bugs with browsers or AT that impact this component, list them here.
+
+Mouse press quick changes to value are not announced to screen readers.
+
```

Comment:
Some known issues.

### Narrator + Edge

1. When no `min` or `max` values are set Narrator announces "minimum 0" and "maximum 0". This is misleading because when these values are not set `SpinButton` does not enforce a min or max value. [Example](https://codesandbox.io/s/sample-spinbutton-zxy72x).

### NVDA + Edge/Chrome

1. When focused on the `SpinButton` input field pressing up/down arrows announces "blank". NVDA + Firefox announces the correct value.